flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java

```diff
@@ -2003,6 +2048,53 @@ public void apply(String key,
 
 	}
 
+	public static class RichSumReducerWithEventTime<W extends Window> extends RichWindowFunction<Tuple2<String, Integer>, Tuple2<String, Integer>, String, W>
+		implements EventTimeFunction<String, W> {
+		private static final long serialVersionUID = 1L;
+
+		private boolean openCalled = false;
+		private static int count = 0;
+		@Override
+		public void open(Configuration parameters) throws Exception {
+			super.open(parameters);
+			openCalled = true;
+		}
+
+		@Override
+		public void close() throws Exception {
+			super.close();
+			closeCalled.incrementAndGet();
+		}
+
+		@Override
+		public void apply(String key,
+						  W window,
+						  Iterable<Tuple2<String, Integer>> input,
```

Comment:
this _should_ fail checkstyle, as the indentation is partly done with spaces. we only use tabs.